ipvs.go

```diff
@@ -0,0 +1,199 @@
+package procfs
+
+import (
+	"bufio"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net"
+	"strconv"
+	"strings"
+)
+
+type IPVSStats struct {
```

Comment:
I think `ProcStat` and `Stat` are called that way because the underlying file is called `stat`. Here it's `ip_vs_stats`, so you could argue that `Stats` makes sense here :)